ui-components-old/src/main/java/com/ivy/old/theme/modal/edit/AmountModal.kt

```diff
@@ -454,23 +457,25 @@ fun KeypadCircleButton(
     )
 }
 
-@SuppressLint("ComposableModifierFactory", "ModifierFactoryExtensionFunction")
+@SuppressLint("ComposableModifierFactory", "ModifierFactoryExtensionFunction", "MissingPermission")
 @Composable
 private fun circleButtonModifier(
     size: Dp = 64.dp,
+    shouldPerformHapticFeedback: Boolean = false,
     onClick: () -> Unit
 ): Modifier {
+    val context = LocalContext.current
+    val vibrator = context.getSystemService(Context.VIBRATOR_SERVICE) as Vibrator
```

Comment:
Use `remember` 
`val vibrator = remember { }`

Reply:
okay

Reply:
@ILIYANGERMANOV are you suggesting something like this:
```
val context = LocalContext.current
val vibrator = remember {context.getSystemService(Context.VIBRATOR_SERVICE) as Vibrator}
```

Reply:
@timmyCoder yes, exactly so it isn't computed of every re-recomposition

Reply:
okay